package.json

```diff
@@ -45,7 +45,7 @@
         "underscore.string": "^3.3.4"
     },
     "devDependencies": {
-        "@exabyte-io/code.js": "2022.9.2-0",
+        "@exabyte-io/code.js": "git+https://github.com/Exabyte-io/code.js.git#0d82444794d72f84865ac0c2623508a0ee9435ee",
```

Comment:
This needs to be a published version

Reply:
true, the corresponding PR is here: https://github.com/Exabyte-io/code.js/pull/35